docs/serverless/08-05-set-external-docker-registry.md

```diff
@@ -9,6 +9,8 @@ By default, you install Kyma with Serverless that uses the internal Docker regis
 - [Google Container Registry (GCR)](https://cloud.google.com/container-registry)
 - [Azure Container Registry (ACR)](https://azure.microsoft.com/en-us/services/container-registry/)
 
+> **CAUTION:** Function images are not cached in the Docker Hub as this registry is not compatible with the caching logic defined in [Kaniko](https://cloud.google.com/cloud-build/docs/kaniko-cache) that Serverless uses for building images. 
```

Comment:
To shorten this one, you could do something like: 
```
> **CAUTION:** Function images are not cached in the Docker Hub. This is because this registry is not compatible with the caching logic defined in [Kaniko](https://cloud.google.com/cloud-build/docs/kaniko-cache) that Serverless uses for building images.
```